internal/cmd/request.go

```diff
@@ -115,3 +123,71 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Println(util.MustToJSONIndent(responseContent))
 	}
 }
+
+// Attempt to convert an API response to a TSV table.
+// Returns false if there are issues or the TSV table if it was successful.
+func PrintCMDResponseTable(response core.APIResponse) (string, bool) {
```

Comment:
Name is misleading.

(I'm purposefully not telling you why the name is deceptive, so you have a chance to think about it.)